### ISSUE TITLE
Uncaught TypeError: Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by shenAlexy** [nikhilkalige/docblockr#262]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.16.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.4
**Thrown From**: [docblockr](https://github.com/nikhilkalige/docblockr) package 0.9.4

### Stack Trace

Uncaught TypeError: Cannot read property 'split' of undefined

```
At /Users/baidu/.atom/packages/docblockr/lib/languages/objc.js:78

TypeError: Cannot read property 'split' of undefined
    at DocsParser.ObjCParser.parse_function (/packages/docblockr/lib/languages/objc.js:78:29)
    at DocsParser.parse (/packages/docblockr/lib/docsparser.js:39:20)
    at DocBlockrAtom.module.exports.DocBlockrAtom.parse_command (/packages/docblockr/lib/docblockr-worker.js:286:29)
    at /packages/docblockr/lib/docblockr-worker.js:42:16)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
     -0:50.2.0 application:open (input.hidden-input)
     -0:20.4.0 docblockr:parse-tab (input.hidden-input)
```

### Non-Core Packages

```
atomic-chrome 0.3.0 
autoupdate-packages 1.3.1 
docblockr 0.9.4 
language-swift 0.5.0 
markdown-pdf 1.5.0 
open-unsupported-files 1.0.19 
seti-icons 1.4.3 
terminal-plus 0.14.5 
```

**Comment by MoritzKn** [comment]
Please document how to reproduce this

**Comment by MoritzKn** [comment]
Fixed by 6b012a071fac26ee0678a2c594a6f07911d0e44f
